cmd/argoexec/commands/root.go

```diff
@@ -105,6 +106,6 @@ func initExecutor() *executor.WorkflowExecutor {
 // checkErr is a convenience function to panic upon error
 func checkErr(err error) {
 	if err != nil {
-		panic(err.Error())
+		panic("InitExecutor failed in sidecar. " + err.Error())
```

Comment:
Can we remove this? prefixing err.Error() with this message doesn't really add value.